test/extended/cli/explain.go

```diff
@@ -480,7 +481,21 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 
 	g.It("should contain proper spec+status for CRDs", func() {
 		crdClient := apiextensionsclientset.NewForConfigOrDie(oc.AdminConfig())
-		crdTypesTest := crdTypes
```

Comment:
Don't we want to update `crdTypes`, wherever we set that up?  If it has any other consumers, I expect they'll want this same handling.